solidity/test/VortexBurner.js

```diff
@@ -0,0 +1,691 @@
+const { accounts, defaultSender, contract } = require('@openzeppelin/test-environment');
+const { expectRevert, expectEvent, BN, constants, balance } = require('@openzeppelin/test-helpers');
+const { expect } = require('../../chai-local');
+const {
+    NATIVE_TOKEN_ADDRESS,
+    registry: {
+        NETWORK_SETTINGS,
+        CONVERTER_FACTORY,
+        CONVERTER_REGISTRY,
+        CONVERTER_REGISTRY_DATA,
+        BANCOR_FORMULA,
+        BANCOR_NETWORK
+    }
+} = require('./helpers/Constants');
+
+const { ZERO_ADDRESS } = constants;
+
+const ContractRegistry = contract.fromArtifact('ContractRegistry');
+const BancorFormula = contract.fromArtifact('BancorFormula');
+const BancorNetwork = contract.fromArtifact('BancorNetwork');
+const DSToken = contract.fromArtifact('DSToken');
+const ConverterRegistry = contract.fromArtifact('TestConverterRegistry');
+const ConverterRegistryData = contract.fromArtifact('ConverterRegistryData');
+const ConverterFactory = contract.fromArtifact('ConverterFactory');
+const StandardPoolConverterFactory = contract.fromArtifact('TestStandardPoolConverterFactory');
+const StandardPoolConverter = contract.fromArtifact('TestStandardPoolConverter');
+const TokenHolder = contract.fromArtifact('TokenHolder');
+const TokenGovernance = contract.fromArtifact('TestTokenGovernance');
+const NetworkSettings = contract.fromArtifact('NetworkSettings');
+const VortexBurner = contract.fromArtifact('VortexBurner');
+
+const PPM_RESOLUTION = new BN(1_000_000);
+
+const TKN = new BN(10).pow(new BN(18));
```

Comment:
`ONE_TOKEN` is more descriptive IMO (especially since we use "TKN" as part of our terminology for some other purpose).
BTW, did you take this from my network-fee tests?

Reply:
`ONE_TOKEN` makes the code below less readable, IMHO.